### PR TITLE
Add favicon and another experimental warning

### PR DESCRIPTION
**Pull request by nickfarrow**

### CODE REVIEW

File: static/index.html [hunk 1]
```diff
@@ -7,13 +7,14 @@
...
 		<link rel="stylesheet" href="style.css">
+		<link rel="icon" type="image/x-icon" href="/pj/static/favicon.ico">
 		<title>nolooking // Lightning PayJoin</title>
 	</head>
 	<body dark-mode>
 		<br>
 		<br>
 		<header class="center">
-			<h1>👻 &nbsp;Lightning PayJoin&nbsp; 👻</h1>
+			<a href="https://nolooking.chaincase.app/" target="_blank" style="text-decoration:none"><h1>👻 &nbsp;Lightning PayJoin&nbsp; 👻</h1></a>
```
Comment (DanGould) [inline static/index.html:17]: Reading this makes me realize "👻 &nbsp;Lightning PayJoin&nbsp; 👻" is messy. Should be <span class="h1">👻 &nbsp;<h1>Lightning PayJoin</h1>&nbsp; 👻</span> for proper semantics with the `--h1` css variables, but it's past halloween anyway to get rid of this.

File: static/index.html [hunk 2]
```diff
@@ -58,6 +59,7 @@ <h2>Queue batches of lightning channels to open in a single transaction</h2>
 				</div>
 				<output id="queued" class="invisible">
 					<h2>PayJoin here to open these channels</h2>
+					<code style="text-align: center;">⚠ This software is still extremely experimental and has not been vetted, use at your own risk ⚠</code>
```
Comment (DanGould) [inline static/index.html:62]: this is not code. best to save semantic elements for semantics and use styles to apply styles

**Review by DanGould — APPROVED** [review]

File: static/index.html (hunk 2 above)
Comment (nickfarrow) [reply static/index.html:62]: True lazy mans HTML 😅

Will resolve

**Comment by DanGould** [comment]
ghost seems seasonal. Can we use the umbrel app icon.svg for favicon?

**Review by 0xBEEFCAF3 — APPROVED** [review]

File: static/index.html (hunk 1 above)
Comment (nickfarrow) [reply static/index.html:17]: Couldn't manage to make this change without getting them onto new lines

**Comment by DanGould** [comment]
Favicons based on umbrel icon: [favicon_package_v0.16.zip](https://github.com/chaincase-app/nolooking/files/9952683/favicon_package_v0.16.zip)

File: static/index.html (hunk 2 above)
Comment (DanGould) [reply static/index.html]: isn't it <code> just `font-family: monospace` ?

Comment (nickfarrow) [reply static/index.html]: It adds border and some other cool styling (put important bits in `.warning` css class)

**Pull request merged**